@xen-orchestra/lite/src/router/index.ts

```diff
@@ -52,4 +61,12 @@ const router = createRouter({
   ],
 });
 
+if (import.meta.env.DEV) {
+  router.addRoute("story", {
+    path: "",
+    name: "story-home",
+    component: () => import("@/views/story/HomeView.vue"),
+  });
+}
```

Comment:
Neither.

This part of code only add a child route to `story` route defined above.

In prod, on "/story" we'll display an empty page with the Story navigation menu on the left.

In dev, it will display the "generator" page (with the navigation too).

Reply:
@pdonias @MathieuRA @Rajaa-BARHTAOUI What's you opinion on this? Does Component Stories be available only in dev mode?

For me, I don't really see the point to be available in production 🤔

Reply:
I don't really see the point of it *not* being available in production. And I see the point of it *being* available in production: if something is broken in production, we can quickly check through the UI what's causing the issue even in production. Would it be an issue to enable it for production?

Reply:
@pdonias I see your point and didn't thought about this. You have convinced me of the usefulness of having it in production ^^